package.json

```diff
@@ -131,8 +131,8 @@
     "moment": "2.19.4",
     "promise-polyfill": "6.1.0",
     "prop-types": "15.6.1",
-    "react": "16.6.3",
-    "react-dom": "16.6.3",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
```

Comment:
we want to use exact versions, so for all of these please remove `^` from the version